index/secondary-index/src/test/scala/org/apache/carbondata/spark/testsuite/secondaryindex/TestSIWithSecondaryIndex.scala

```diff
@@ -577,6 +589,59 @@ class TestSIWithSecondaryIndex extends QueryTest with BeforeAndAfterAll {
     assert(ex.getMessage.contains("Problem loading data while creating secondary index:"))
   }
 
+  test("test SI with carbon.use.local.dir as true") {
+    CarbonProperties.getInstance()
```

Comment:
Note default for `CARBON_LOADING_USE_YARN_LOCAL_DIR` is `true`. Why do we set to `true` again both at the begin and end of testcase?

Reply:
When some testcase has set this value and not reverted back to default, we need to add this property again.

Reply:
@nihal0107 if this property is not used anywhere, then you remove it

Reply:
wanted to add for `carbon.use.local.dir` as  `false`. Made the changes.